CondTools/RunInfo/src/LHCInfoPerFillPopConSourceHandler.cc

```diff
@@ -520,12 +520,8 @@ LHCInfoPerFillPopConSourceHandler::executeLumiQuery(const cond::OMSService& oms,
   query->filterEQ("beams_stable", "true");
   query->limit(cond::lhcInfoHelper::kLumisectionsQueryLimit);
 
-  bool executed = query->execute();
-  if (executed) {
-    return std::make_tuple(query->result(), true, std::move(query));
-  } else {
-    return std::make_tuple(cond::OMSServiceResult(), false, std::move(query));
-  }
+  bool success = query->execute();
+  return { (success ? query->result() : cond::OMSServiceResult()), success, std::move(query) };
```

Comment:
Why this change of the return type? Does C++ automatically cast into the return type of the method?

Reply:
Yes, this is a brace-initialization / list-initialization and this way the tuple gets initialized in-place